recipe/gen_patch_json.py

```diff
@@ -554,6 +554,11 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
                 i = record['depends'].index('zstandard')
                 record['depends'][i] = 'zstandard <0.15'
 
+        if record_name == "pytorch" and record["version"] == "1.13.1" and record.get('timestamp', 0) < 1675431752816:
```

Comment:
```suggestion
        if record_name == "pytorch" and record["version"] == "1.13.1" and record['build_number'] == 0 and record.get('timestamp', 0) < 1675431752816:
```

Reply:
I know it is overly cautious

Reply:
There's literally no other build that could ever satisfy both the version _and_ time constraint, but if you want...

Reply:
I just can't read these time constraints. they mean nothing to me.

Reply:
If we asked for the output of 
```
print(datetime.utcnow())
2023-02-03 03:57:55.382143
```
then that would be alot better IMO.

Reply:
> I just can't read these time constraints. they mean nothing to me.

Big same

> If we asked for the output of [...] then that would be alot better IMO.

Just today I was thinking of writing a convenience function in that module that converts between a human-readable timestamp and a unix epoch int that the `record.timestamp` needs. Would that be interesting you think?

Reply:
i just don't like repo patches alltogether ^_^ but yes. I would rather have human readable timestamds with +0:00 to indicate greenwich time